lti_lib/register_platform.js

```diff
@@ -0,0 +1,53 @@
+const mongoose = require('mongoose');
+const Database = require('./mongoDB/Database.js');
+const { keyGenerator } = require('./keyGenerator.js');
+const Schema = mongoose.Schema;
+
+const platformSchema = new Schema({
+  consumerUrl: String,
+  consumerName: String,
+  consumerClientID: String,
```

Comment:
the ClientID is the id that the Consumer assigns to the Tool, correct?  If so, I think it would be clearer if you add 'Tool' to the name of the field - either toolClientID or consumerToolClientID